### PR TITLE
Add news post for release 20170421

### PR DESCRIPTION
**Pull request by obrakmann**
I was lazy and copied loads of text from the first playtest's news post.

### CODE REVIEW

File: content/news/release-20170421.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<div style="text-align:center" markdown="1">
+![New Dune 2000 missions](/images/news/20170421-d2k-new-missions.png)
+
+Dune 2000 now comes with 13 campaign missions.
+</div>
+
+<div style="text-align:center" markdown="1">
+![New Gap Generator behaviour](/images/news/20170421-ra-gapgen.png)
+
+The re-written behaviour of the gap generator more closely resembles that of the original game.
```
Comment (pchote) [inline content/news/release-20170421.md]: I'm reasonably sure that it actually doesn't.

File: content/news/release-20170421.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+The in-game multiplayer server list will now list games for *all* the mods that a player has installed (even across multiple installations), and will allow players to directly switch to another mod when joining one.  This applies across versions as well as mods, meaning that future OpenRA releases will be able to coexist, and switching between release and playtest servers will be just as easy as switching between RA and TD servers today.
+
+<div style="text-align:center" markdown="1">
+![New MP server list switcher](/images/news/20170304-mpmodswitch.png)
+
+The multiplayer server list is now smarter about multiple OpenRA installations.
+</div>
+
+The next release (aiming for mid/late 2017) will complete the mod-support transition by removing the in-game mod chooser and support for manually installed mod packages: all mods (both official and community) will be listed individually with their own names and icons in your Start Menu / Dock / Launcher.
```
Comment (pchote) [inline content/news/release-20170421.md]: I very much doubt that "mid" 2017 is practical anymore.

**Comment by obrakmann** [comment]
Updated.

**Comment by obrakmann** [comment]
Updated with changelog link and release id.

**Pull request merged**